mirrord/operator/src/crd.rs

```diff
@@ -161,6 +301,8 @@ pub struct CopyTargetSpec {
     /// Should the operator scale down target deployment to 0 while this pod is alive.
     /// Ignored if [`Target`] is not [`Target::Deployment`].
     pub scale_down: bool,
+    /// queue id -> (attribute name -> regex)
+    pub sqs_filter: Option<HashMap<QueueId, SqsMessageFilter>>,
```

Comment:
Maybe we should use a nested struct here? To group all queue filters in a separate type

Reply:
Should I just send over the whole `SplitQueuesConfig`, and do the partitioning by queue types in the operator's side?

Reply:
Yeah, sounds good